frontend/src/logic/schedule-parser/shifts-info.parser.ts

```diff
@@ -53,9 +54,6 @@ export class ShiftsInfoParser {
   private fillRowWithShifts(row: string[]): Shift[] {
     let previousShift: Shift = null;
     return row.map((i) => {
-      if (typeof i == "number") {
```

Comment:
Row jest tablicą stringów, więc "i" jej obiekt nigdy nie będzie typu "number.

Reply:
Dodałem to, ponieważ przy parsowaniu mogły pojawić się liczby ( na przykład godziny pracy ). Ale i tak raczej będą stringiem, bo są sczytane z harmonogramu. Więc ta zmiana jaknajbardziej ma sens